azkaban-common/src/main/java/azkaban/project/DirectoryFlowLoader.java

```diff
@@ -114,7 +115,7 @@ public DirectoryFlowLoader(final Props props) {
   /**
    * Loads all project flows from the directory.
    *
-   * @param project The project.
+   * @param project    The project.
```

Comment:
Is this intentional?

Reply:
AFAIK I have the IDEA Save Actions plugin configured as documented in https://github.com/azkaban/azkaban/blob/master/CONTRIBUTING.md#style-guides

But if this auto formatting doesn't look right, could you advise on how to change the Save Actions conf?

Reply:
Regarding formatting, I have just added a commit to "Set hard wrap at 100". It seemed to work to keep the same formatting for some of the existing javadocs in FlowRunner.java. On the other hand some code that I didn't touch got split to 2 lines 🤔

In general it would be great to just apply some plugin on build time to reformat the code so that the result would be the same even if someone commits code without all the right IDE plugins & settings.